handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+- name: restart certbot
```

Comment:
Is this handler being used anywhere? "restart certbot" seems a bit misleading. I'm assuming this command is what generates certs.